Doc/faq/programming.rst

```diff
@@ -1741,11 +1741,34 @@ but effective way to define class private variables.  Any identifier of the form
 is textually replaced with ``_classname__spam``, where ``classname`` is the
 current class name with any leading underscores stripped.
 
+More generally, the corresponding private member is accessed by its defining
```

Comment:
Do we need this extra paragraph? I'm conscious that when changing docs, we're always likely to feel that more documentation is needed, leading to the docs becoming more and more verbose over time. This FAQ entry is slightly imprecise relative to the language reference, but that's fine; it's not clear to me that the extra paragraph is necessary.

Reply:
Mmh, maybe not but I think the example should be kept (it's a FAQ and generally examples in such cases are much more easy to understand). I admit that the paragraph could be more confusing than helping.

Reply:
How about replacing this paragraph with "The identifier can be used unchanged within the class, but to access it outside the class, the mangled name must be used:", followed by the example?

Reply:
Oh sure. Actually, I wanted to entirely suppress the paragraph and just show the example, but I can add this introductory sentence!